docs/src/api.md

```diff
@@ -227,12 +227,12 @@ Notice that schedulers can be given directly to model creation, and thus become
 ### Predefined schedulers
 Some useful schedulers are available below as part of the Agents.jl public API:
 ```@docs
-fastest
-by_id
-random_activation
-partial_activation
-property_activation
-by_type
+schedule_fastest
+schedule_by_id
+schedule_randomly
+schedule_partial
+schedule_property
```

Comment:
Could you rename these two to `schedule_partially` and `schedule_by_property`? I think these names are more sensible...

Reply:
Updated.